precompiles/bank/logs.go

```diff
@@ -0,0 +1,46 @@
+package bank
+
+import (
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/vm"
+
+	"github.com/zeta-chain/node/precompiles/logs"
+)
+
+func (c *Contract) addEventLog(
+	ctx sdk.Context,
```

Comment:
consider having an unexported struct as an input. This method has too many args

Reply:
Created `eventData` struct for this case.